README.md

```diff
@@ -1,23 +1,27 @@
-# Riskbook's clone of haskell-xmpp
+[![Maintenance badge](https://img.shields.io/maintenance/yes/2020)](https://riskbook.com/)
 
-Fixes:
+Fully functional haskell-xmpp bindings used in production
+at riskbook.
+This integrates well with ejabbered for example.
+
+Haskell XMPP (eXtensible Message Passing Protocol, a.k.a. Jabber) library.
+Unlike package network-protocol-xmpp, which uses libxml-sax, this library uses HaXml and supports MUC.
+However, MUC support of the moment is worse than that in package XMPP.
+This library make extensive use of STM and threads to simplify writing message-handling code.
+
+As of version 2 riskbook maintains this.
+We patched the orignal work by dmitry with these fixes:
 - [x] Duplicate Show instances
 - [x] Missing FlexibleContexts
 - [x] GADTs for Stanzas
 
-Original README contents follow:
-*** 
-Attempt to write an XMPP bindings for haskell
-
-Alpha-version/work in progress
+## Contributing
+Feel free to submit any PR or issue.
 
-To build, obtain and install latest HaXml 
-(darcs get http://www.cs.york.ac.uk/fp/darcs/HaXml)
-, then use supplied rebuild.sh
+Questions can be send to: jappie@riskbook.com
 
-Questions or proposals?
-Contact dastapov at gmail.com (via mail),
-adept at jabber.kiev.ua (via XMPP :)
-or adept at #haskell (via IRC)
+## License
+MIT
```

Comment:
This is not what it says in the cabal file.
Also upgrading licence require at least note with the years and copyrights of the previous license holders 🙂

Reply:
Good catch!

Reply:
Note, that it needs to be added not only to new files but to the old one as well, where we need to specify the previous license copyright + our new one.

Reply:
I added, please check if done right, I've no idea.